Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -195,8 +195,13 @@ def self.run_checks(
       extract_plist = true if formulae_and_casks_total == 1
 
       formulae_checked = formulae_and_casks_to_check.map.with_index do |formula_or_cask, i|
-        formula = formula_or_cask if formula_or_cask.is_a?(Formula)
-        cask = formula_or_cask if formula_or_cask.is_a?(Cask::Cask)
+        case formula_or_cask
+        when Formula
+          formula = formula_or_cask
+          formula.head&.downloader&.quiet!
```

Comment:
Question: This just sets the HEAD downloader to quiet mode, right? It doesn't seem to actually download anything so moving it here seems fine.

Reply:
That matches my understanding